### ISSUE TITLE
Building branch 0.6.0 with /MD

### ISSUE THREAD

**Issue by filip-svoboda** [IfcOpenShell/IfcOpenShell#559]
Hi,
I'm trying to build branch 0.6.0 dynamically, but I'm getting a lot of linker errors. I'm able to build master branch with /MD more or less according to these suggestions:  https://github.com/IfcOpenShell/IfcOpenShell/issues/324, but it seems that there are problems with changed geom libraries.

Any suggestions?
Thanks
![image](https://user-images.githubusercontent.com/35653184/54126771-9e6ecf00-4408-11e9-96bd-0ec50e4fe753.png)

**Comment by aothms** [comment]
Indeed. This is one of the main reasons why master is still on 0.5. Help here is much appreciated. We need to check if all symbols are prefixed with the correct declaration. E.g. the `IFC_PARSE_API` here in https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.6.0/src/ifcparse/IfcFile.h#L38 and making sure the correct parameters are passed from cmake https://github.com/IfcOpenShell/IfcOpenShell/blob/master/cmake/CMakeLists.txt#L592 For IfcGeom this might need to be more granular because of the added libraries.

Not sure if all libraries have to be dynamic though. In 0.7 there's even a lot more libraries due to the addition of CGAL. It might make sense for some of them to remain static. If only IfcParse and IfcGeom are in fact dynamic, things might still work without changes to the code base.

**Comment by BIMvoice** [comment]
Status: this was filed against the 0.6-era build where shared linking was known-broken ("master is still on 0.5"). On current `v0.8.0` shared builds are routine — every CI wheel and conda package links the wrapper as a shared library, and `cmake/FindIfcOpenShell.cmake` plus `IfcOpenShellConfig.cmake.in` now exist for downstream consumption. The 2019 problem statement no longer matches the build system; recommend closing as obsolete.
